nginx.tpl

```diff
@@ -1,3 +1,10 @@
+{% for srvname, service in services.items() %}
```

Comment:
Could just iterate once and print the upstream + server pair together? Might make it a bit nicer to read.